### DISCUSSION TITLE
Query regarding using Blackduck/Coverity scan

### DISCUSSION THREAD

**Discussion by vrukesh** [picolibc/picolibc#887] [General]
Hi Keith,
Hope you are doing well !
Wondering if picolibc does any Blackduck/Coverity scan?
Github is already linked with Blackduck/Coverity scan, and LLVM project already is present here (results are stale):
https://scan.coverity.com/projects/llvm?tab=overview

If not already, is it fine to add picolibc into Blackduck/Coverity scan?

Thanks and Regards,
Vrukesh

**Comment by keith-packard** [comment]
Sure, more scanning seems like a fine plan.

**Reply by vrukesh** [reply]
Hi Keith,
Thanks for quick response !
I have added picolibc project into https://scan.coverity.com/ , it will take upto 48 hours to appear.

Since this is first time enablement for scanning picolibc, I have selected: "Access to summary and defects requires Project Maintainer approval".

Thanks and Regards,
Vrukesh

**Comment by vrukesh** [comment]
Hi Keith,
Due to some unknown reason, picolibc project could not be added in Coverity Blackduck scan.
I have added it again now, and will keep you posted once the project is available.
Regards,
Vrukesh

**Comment by vrukesh** [comment]
Hi Keith,

Blackduck Coverity scan for picolibc is configured now: https://scan.coverity.com/projects/picolibc?tab=overview
Scan defects: https://scan9.scan.coverity.com/#/project-view/61422/15896

Hope that the project is configured properly.

It would be nice (not urgent) to make a plan for handling these 161 defects.

Possibly since I added picolibc for scan, am showing up as admin. I will be happy to change the admin ownership back to you. Kindly let know if I can initiate this with Coverity team.

Thanks and Regards,
Vrukesh
